parsec/backend/invite.py

```diff
@@ -201,8 +203,16 @@ def generate_invite_email(
             use_ssl=backend_addr.use_ssl,
         )
     )
+    http_invite_link = parsec_invite_link.replace("parsec://", "")
+    split_invite_link = http_invite_link.split("/")
+    http_invite_link = "http://" + split_invite_link[0] + "/api/redirect/" + split_invite_link[1]
```

Comment:
How do we now it's `https` ?

(We should use the `backend.config.backend_addr` variable to know which scheme to use)